internal/dynamo/client.go

```diff
@@ -87,6 +86,31 @@ func (c *Client) OneByUID(ctx context.Context, uid string, v interface{}) error
 	return attributevalue.UnmarshalMap(response.Items[0], v)
 }
 
+func (c *Client) AllScheduledEventsByUID(ctx context.Context, uid string, v interface{}) error {
+	response, err := c.svc.Query(ctx, &dynamodb.QueryInput{
+		TableName: aws.String(c.table),
+		IndexName: aws.String(lpaUIDIndex),
+		ExpressionAttributeNames: map[string]string{
+			"#LpaUID": "LpaUID",
+			"#SK":     "SK",
+		},
+		ExpressionAttributeValues: map[string]types.AttributeValue{
+			":LpaUID": &types.AttributeValueMemberS{Value: uid},
+			":SK":     &types.AttributeValueMemberS{Value: scheduledPrefix},
```

Comment:
would probably be worth appending a `#` on this

Reply:
would it be worth making this less specific as `AllByUIDAndPartialSK` or something?

Reply:
Yeah fair point 👍